--- codecs/Makefile.libplc10	2011-02-23 00:04:49.000000000 +0100
+++ codecs/Makefile	2012-07-19 11:15:47.695540978 +0200
@@ -17,8 +17,8 @@
 MENUSELECT_CATEGORY=CODECS
 MENUSELECT_DESCRIPTION=Codec Translators
 
-LIBILBC:=ilbc/libilbc.a
-LIBLPC10:=lpc10/liblpc10.a
+LIBILBC:=-lilbc
+LIBLPC10:=-llpc10
 
 all: _all
 
@@ -32,8 +32,6 @@
 
 clean::
 	$(MAKE) -C gsm clean
-	$(MAKE) -C lpc10 clean
-	$(MAKE) -C ilbc clean
 	rm -f g722/*.[oa]
 	rm -f speex/*.[oa]
 
@@ -41,15 +39,8 @@
 	@mkdir -p gsm/lib
 	@$(MAKE) -C gsm lib/libgsm.a
 
-$(LIBLPC10):
-	@$(MAKE) -C lpc10 all
-
 $(if $(filter codec_lpc10,$(EMBEDDED_MODS)),modules.link,codec_lpc10.so): $(LIBLPC10)
 
-$(LIBILBC):
-	@$(MAKE) -C ilbc all _ASTCFLAGS="$(filter-out -Wmissing-prototypes -Wmissing-declarations -Wshadow,$(_ASTCFLAGS)) $(AST_NO_STRICT_OVERFLOW)"
-
-
 $(if $(filter codec_ilbc,$(EMBEDDED_MODS)),modules.link,codec_ilbc.so): $(LIBILBC)
 
 $(if $(filter codec_g722,$(EMBEDDED_MODS)),modules.link,codec_g722.so): g722/g722_encode.o g722/g722_decode.o
