diff -Naurp asterisk-11.5.0/configure.ac asterisk-11.5.0.oden/configure.ac
--- asterisk-11.5.0/configure.ac	2013-08-06 14:52:53.388690216 +0200
+++ asterisk-11.5.0.oden/configure.ac	2013-08-06 14:52:29.890518007 +0200
@@ -1908,7 +1908,7 @@ AST_EXT_LIB_CHECK([NBS], [nbs], [nbs_con
 AST_EXT_TOOL_CHECK([NEON], [neon-config])
 
 AST_EXT_TOOL_CHECK([NEON29], [neon-config], , [--libs],
-[#include <ne_auth.h>],
+[#include <neon/ne_auth.h>],
 [#ifndef NE_AUTH_NTLM
 #error Need libneon >= 0.29.0
 #endif])
diff -Naurp asterisk-11.5.0/res/res_calendar_caldav.c asterisk-11.5.0.oden/res/res_calendar_caldav.c
--- asterisk-11.5.0/res/res_calendar_caldav.c	2012-05-18 16:43:44.000000000 +0200
+++ asterisk-11.5.0.oden/res/res_calendar_caldav.c	2013-08-06 14:52:29.890518007 +0200
@@ -32,11 +32,11 @@
 ASTERISK_FILE_VERSION(__FILE__, "$Revision: 366917 $")
 
 #include <libical/ical.h>
-#include <ne_session.h>
-#include <ne_uri.h>
-#include <ne_request.h>
-#include <ne_auth.h>
-#include <ne_redirect.h>
+#include <neon/ne_session.h>
+#include <neon/ne_uri.h>
+#include <neon/ne_request.h>
+#include <neon/ne_auth.h>
+#include <neon/ne_redirect.h>
 #include <libxml/xmlmemory.h>
 #include <libxml/parser.h>
 
diff -Naurp asterisk-11.5.0/res/res_calendar_ews.c asterisk-11.5.0.oden/res/res_calendar_ews.c
--- asterisk-11.5.0/res/res_calendar_ews.c	2012-10-31 15:49:33.000000000 +0100
+++ asterisk-11.5.0.oden/res/res_calendar_ews.c	2013-08-06 14:52:29.891517977 +0200
@@ -29,15 +29,15 @@
 
 ASTERISK_FILE_VERSION(__FILE__, "$Revision: 375532 $")
 
-#include <ne_request.h>
-#include <ne_session.h>
-#include <ne_uri.h>
-#include <ne_socket.h>
-#include <ne_auth.h>
-#include <ne_xml.h>
-#include <ne_xmlreq.h>
-#include <ne_utils.h>
-#include <ne_redirect.h>
+#include <neon/ne_request.h>
+#include <neon/ne_session.h>
+#include <neon/ne_uri.h>
+#include <neon/ne_socket.h>
+#include <neon/ne_auth.h>
+#include <neon/ne_xml.h>
+#include <neon/ne_xmlreq.h>
+#include <neon/ne_utils.h>
+#include <neon/ne_redirect.h>
 
 #include "asterisk/module.h"
 #include "asterisk/calendar.h"
diff -Naurp asterisk-11.5.0/res/res_calendar_exchange.c asterisk-11.5.0.oden/res/res_calendar_exchange.c
--- asterisk-11.5.0/res/res_calendar_exchange.c	2012-05-18 16:43:44.000000000 +0200
+++ asterisk-11.5.0.oden/res/res_calendar_exchange.c	2013-08-06 14:52:29.891517977 +0200
@@ -32,11 +32,11 @@
 ASTERISK_FILE_VERSION(__FILE__, "$Revision: 366917 $")
 
 #include <libical/ical.h>
-#include <ne_session.h>
-#include <ne_uri.h>
-#include <ne_request.h>
-#include <ne_auth.h>
-#include <ne_redirect.h>
+#include <neon/ne_session.h>
+#include <neon/ne_uri.h>
+#include <neon/ne_request.h>
+#include <neon/ne_auth.h>
+#include <neon/ne_redirect.h>
 #include <iksemel.h>
 
 #include "asterisk/module.h"
diff -Naurp asterisk-11.5.0/res/res_calendar_icalendar.c asterisk-11.5.0.oden/res/res_calendar_icalendar.c
--- asterisk-11.5.0/res/res_calendar_icalendar.c	2013-01-30 15:15:27.000000000 +0100
+++ asterisk-11.5.0.oden/res/res_calendar_icalendar.c	2013-08-06 14:52:29.891517977 +0200
@@ -31,11 +31,11 @@
 ASTERISK_FILE_VERSION(__FILE__, "$Revision: 380452 $")
 
 #include <libical/ical.h>
-#include <ne_session.h>
-#include <ne_uri.h>
-#include <ne_request.h>
-#include <ne_auth.h>
-#include <ne_redirect.h>
+#include <neon/ne_session.h>
+#include <neon/ne_uri.h>
+#include <neon/ne_request.h>
+#include <neon/ne_auth.h>
+#include <neon/ne_redirect.h>
 
 #include "asterisk/module.h"
 #include "asterisk/calendar.h"
