diff -ur configure.ac asterisk-1.8.1.1/configure.ac
--- configure.ac.neon	2010-12-17 12:04:52.000000000 +0100
+++ configure.ac	2010-12-17 12:05:05.000000000 +0100
@@ -1687,7 +1687,7 @@
 AST_EXT_TOOL_CHECK([NEON], [neon-config])
 
 AST_EXT_TOOL_CHECK([NEON29], [neon-config], , [--libs],
-[#include <ne_auth.h>],
+[#include <neon/ne_auth.h>],
 [#ifndef NE_AUTH_NTLM
 #error Need libneon >= 0.29.0
 #endif])
--- res/res_calendar_caldav.c.neon	2011-07-14 22:13:06.000000000 +0200
+++ res/res_calendar_caldav.c	2011-09-04 11:24:24.283801326 +0200
@@ -32,11 +32,11 @@
 ASTERISK_FILE_VERSION(__FILE__, "$Revision: 366881 $")
 
 #include <libical/ical.h>
-#include <ne_session.h>
-#include <ne_uri.h>
-#include <ne_request.h>
-#include <ne_auth.h>
-#include <ne_redirect.h>
+#include <neon/ne_session.h>
+#include <neon/ne_uri.h>
+#include <neon/ne_request.h>
+#include <neon/ne_auth.h>
+#include <neon/ne_redirect.h>
 #include <libxml/xmlmemory.h>
 #include <libxml/parser.h>
 
--- res/res_calendar_ews.c.neon	2011-07-14 22:13:06.000000000 +0200
+++ res/res_calendar_ews.c	2011-09-04 11:26:31.751491087 +0200
@@ -29,15 +29,15 @@
 
 ASTERISK_FILE_VERSION(__FILE__, "$Revision: 366741 $")
 
-#include <ne_request.h>
-#include <ne_session.h>
-#include <ne_uri.h>
-#include <ne_socket.h>
-#include <ne_auth.h>
-#include <ne_xml.h>
-#include <ne_xmlreq.h>
-#include <ne_utils.h>
-#include <ne_redirect.h>
+#include <neon/ne_request.h>
+#include <neon/ne_session.h>
+#include <neon/ne_uri.h>
+#include <neon/ne_socket.h>
+#include <neon/ne_auth.h>
+#include <neon/ne_xml.h>
+#include <neon/ne_xmlreq.h>
+#include <neon/ne_utils.h>
+#include <neon/ne_redirect.h>
 
 #include "asterisk/module.h"
 #include "asterisk/calendar.h"
--- res/res_calendar_exchange.c.neon	2011-07-14 22:13:06.000000000 +0200
+++ res/res_calendar_exchange.c	2011-09-04 11:27:40.311552611 +0200
@@ -32,11 +32,11 @@
 ASTERISK_FILE_VERSION(__FILE__, "$Revision: 366881 $")
 
 #include <libical/ical.h>
-#include <ne_session.h>
-#include <ne_uri.h>
-#include <ne_request.h>
-#include <ne_auth.h>
-#include <ne_redirect.h>
+#include <neon/ne_session.h>
+#include <neon/ne_uri.h>
+#include <neon/ne_request.h>
+#include <neon/ne_auth.h>
+#include <neon/ne_redirect.h>
 #include <iksemel.h>
 
 #include "asterisk/module.h"
--- res/res_calendar_icalendar.c.neon	2011-07-14 22:13:06.000000000 +0200
+++ res/res_calendar_icalendar.c	2011-09-04 11:28:32.447881286 +0200
@@ -31,11 +31,11 @@
 ASTERISK_FILE_VERSION(__FILE__, "$Revision: 366881 $")
 
 #include <libical/ical.h>
-#include <ne_session.h>
-#include <ne_uri.h>
-#include <ne_request.h>
-#include <ne_auth.h>
-#include <ne_redirect.h>
+#include <neon/ne_session.h>
+#include <neon/ne_uri.h>
+#include <neon/ne_request.h>
+#include <neon/ne_auth.h>
+#include <neon/ne_redirect.h>
 
 #include "asterisk/module.h"
 #include "asterisk/calendar.h"
